.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
Not important, but good to add any IDE or editor ignores to your own global gitignore. People use so many different tools, and it saves putting them all into the project gitignore.